docs/src/main/asciidoc/kafka.adoc

```diff
@@ -384,14 +384,45 @@ The `io.smallrye.reactive.messaging.annotations.Emitter`, `io.smallrye.reactive.
 The new `Emitter.send` method returns a `CompletionStage` completed when the produced message is acknowledged.
 ====
 
-== Kafka Health Check
+== Kafka Health Checks
 
-If you are using the `quarkus-smallrye-health` extension, `quarkus-kafka` can add a readiness health check
-to validate the connection to the broker. This is disabled by default.
+Quarkus provides several health checks for Kafka.
+These checks are used in combination with the `quarkus-smallrye-health` extension.
 
-If enabled, when you access the `/q/health/ready` endpoint of your application you will have information about the connection validation status.
+When using the `quarkus-kafka` extension, you can enable _readiness_ health check by setting the `quarkus.kafka.health.enabled` property to `true` in your `application.properties`.
+This check reports the status of the interaction with a _default_ Kafka broker (configured using `kafka.bootstrap.servers`).
+That check requires an _admin connection_ with the Kafka broker.
```

Comment:
I'm surprised you need elevated privileges for a simple thing like health checks? Or did I misunderstand what an admin connection is?

Reply:
In general, user credentials are enough to create a Kafka Admin client, and so execute the check. That being said I agree with you, it's like requiring a privileged account on a database to check if the database is present (if `DROP TABLE USERS` succeeds that means we are connected to the database).

That's why in reactive messaging, we moved away from that model and we have been discussing multiple alternatives. The new one (in Reactive Messaging 2.8) looks good so far and does not require an admin connection anymore. But, I would like to see if we get more reports (we had one but it was actually correct, the application was not consistent). If we don't get more reports we will apply the same strategy in the kafka-client extension. 

Also, the motivation for this is that not all Kafka deployments will allow admin connections using user credentials.